R/commonutils.R

```diff
@@ -1,3 +1,27 @@
+#' Check if a R package is installed.
+#' If installed, load the library;
+#' otherise, stop and issue warning message
+#'
+#' @param pkgNmae Package Name, a string
+#' @param level Message levels: \cr
+#' 1: message \cr
+#' 2: warn \cr
+#' 3: stop \cr
+#' @return A boolean, if the package exists.
+#' @export
+checkPackage <- function(pkgName, level) {
```

Comment:
Oh wait. You are just loading a package conditionally here. Why not simply use `require`. It will return an error message if it can't find the package. What are we trying to accomplish here?